### PR TITLE
[DE] Add `Help_centre/Website`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
The first point that I wanted to mention is that I'm not sure about the translation of `me! page`. The English wiki page uses `Me! page`, whereas the website has `me!` and `ich!` for English and German respectively. For now, I've used `ich!-Seite`.

The second thing is that I'm unsure about the required commas in line 92, especially around `mit dem was du aushängen möchtest`. Maybe we should part this in two sentences, which would also improve readability.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Help_centre/Website/de.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+Diese Sektion befasst sich mit Problemen auf der osu!-Webseite.
+
+## Häufige Probleme
+
+### Ich habe keine Antwort auf mein Support-Ticket erhalten und es sind schon über 2 Wochen vergangen!
+
+**In der überwiegenden Mehrheit der Fälle liegt das an einer hohen Zahl an Tickets, die zur gleichen Zeit in unser System eingehen.**
+
+Du kannst immer eine einzelne Antwort auf dein aktuell offenes Ticket senden, um den Status deines Tickets anzufragen, aber stelle sicher, dass du dies nicht mehr als zwei- oder dreimal pro Woche machst. Ebenfalls kannst du uns auf Twitter unter [@osusupport](https://twitter.com/osusupport) kontaktieren, um nach deinem Ticket zu fragen.
```
Comment (respektive) [inline wiki/Help_centre/Website/de.md:32]:
```suggestion
Du kannst immer eine einzelne Antwort auf dein aktuell offenes Ticket senden, um den Status deines Tickets anzufragen, aber stelle sicher, dass du dies nicht mehr als ein- oder zweimal pro Woche machst. Ebenfalls kannst du uns auf Twitter unter [@osusupport](https://twitter.com/osusupport) kontaktieren, um nach deinem Ticket zu fragen.
```

File: wiki/Help_centre/Website/de.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+#### Verhindern, dass dich neue private Nachrichten aus dem Forum erreichen
+
+Gehe zu deinen [Account-Einstellungen](https://osu.ppy.sh/home/account/edit) und klicke auf `blockiere nachrichten von benutzern, die nicht auf deiner freundesliste sind` in der Sektion Privatsphäre. Private Nachrichten von Benutzern in deiner Freundesliste werden dich trotzdem erreichen.
+
+### Was ist, wenn ich nach dem Blockieren eines Nutzers immer noch Probleme mit ihm habe?
+
+**Keine Angst, unsere Support-Mitarbeiter werden dir helfen.**
+
+Falls dich ein Benutzer immer noch auf andere Weise belästigt, seine Freunde oder andere benutzt, um deine Blockierung zu umgehen, sende uns bitte eine E-Mail an [support@ppy.sh](mailto:support@ppy.sh).
```
Comment (respektive) [inline wiki/Help_centre/Website/de.md:74]:
```suggestion
Falls dich ein Benutzer immer noch auf andere Weise belästigt, seine Freunde oder andere Personen benutzt, um deine Blockierung zu umgehen, sende uns bitte eine E-Mail an [support@ppy.sh](mailto:support@ppy.sh).
```

File: wiki/Help_centre/Website/de.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+Auch wenn dein [Supporter-Tag](https://osu.ppy.sh/home/support) abläuft, wird deine ich!-Seite **sichtbar bleiben** und du kannst den Inhalt immer noch verändern.
+
+### Die Inhalte meiner ich!-Seite fehlen!
+
+**Das kann passieren, wenn deine ich!-Seite unangebrachten Inhalt hatte, gemäß unseren [Community-Regeln](/wiki/Rules).**
+
+In solchen Fällen darfst du beim Support-Team ein Ticket öffnen, indem du eine E-Mail an [accounts@ppy.sh](mailto:accounts@ppy.sh) sendest und deine Situation mit dem Team diskutierst.
+
+Solltest du zu irgendeinem Zeitpunkt Zweifel haben, ob etwas angemessen für eine ich!-Seite ist, darfst du einem Mitglied des [globalen Moderationsteams](/wiki/People/The_Team/Global_Moderation_Team) eine Nachricht senden, mit dem was du aushängen möchtest, und sie werden dich wissen lassen, ob es in Ordnung ist.
```
Comment (respektive) [inline wiki/Help_centre/Website/de.md:92]: While being a bit long, this generally reads fine to me, but if you wanna make it into two sentences that's also fine.

File: wiki/Help_centre/Website/de.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+### Muss ich meine E-Mail-Adresse für die Registrierung verwenden?
+
+**Ja. Wir benötigen deine E-Mail-Adresse, um dir bei Bedarf Anfragen für Passwortrücksetzungen und Bestätigungscodes zu schicken.**
+
+Wenn du Probleme hast, dich über den Spiel-Client zu verbinden, überprüfe bitte die Einstellungen der Firewall und der Antivirensoftware, um sicherzugehen, dass osu! nicht blockiert wird.
+
+Falls du jemals das Passwort verlierst oder nicht auf deinen Account zugreifen kannst, ist deine Registrierungs-E-Mail unsere erste Anlaufstelle für uns.
```
Comment (respektive) [inline wiki/Help_centre/Website/de.md:100]:
```suggestion
Falls du jemals das Passwort verlierst oder nicht auf deinen Account zugreifen kannst, ist deine Registrierungs-E-Mail unsere erste Anlaufstelle für dich.
```

**Review by respektive — CHANGES_REQUESTED** [review]

File: wiki/Help_centre/Website/de.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+
+## Häufige Probleme
+
+### Ich habe keine Antwort auf mein Support-Ticket erhalten und es sind schon über 2 Wochen vergangen!
+
+**In der überwiegenden Mehrheit der Fälle liegt das an einer hohen Zahl an Tickets, die zur gleichen Zeit in unser System eingehen.**
+
+Du kannst immer eine einzelne Antwort auf dein aktuell offenes Ticket senden, um den Status deines Tickets anzufragen, aber stelle sicher, dass du dies nicht mehr als zwei- oder dreimal pro Woche machst. Ebenfalls kannst du uns auf Twitter unter [@osusupport](https://twitter.com/osusupport) kontaktieren, um nach deinem Ticket zu fragen.
+
+In seltenen Umständen können Support-Tickets von <!--a-->chronisch straffälligen Nutzern absichtlich ignoriert werden, jedoch wird sich das [Support-Team](/wiki/People/The_Team/Account_support_team) vorher **immer** bei dir melden, wenn sie nicht mehr länger antworten werden. Falls du noch keine Benachrichtigung wie diese erhalten hast, brauchst du dir keine Sorgen machen!
```
Comment (The-Last-Cookie) [inline wiki/Help_centre/Website/de.md:34]:
```suggestion
In seltenen Umständen können Support-Tickets von chronisch straffälligen Nutzern absichtlich ignoriert werden, jedoch wird sich das [Support-Team](/wiki/People/The_Team/Account_support_team) vorher **immer** bei dir melden, wenn sie nicht mehr länger antworten werden. Falls du noch keine Benachrichtigung wie diese erhalten hast, brauchst du dir keine Sorgen machen!
```
Wanted to check on `chronisch straffällig` once again, as I've always heard `chronisch` in the context of illnesses and rarely/never in such a case. I can't find a better translation for this though and deepl suggests the same.

Comment (respektive) [reply wiki/Help_centre/Website/de.md:34]: There is the term `chronic offender` which could be translated to `Gewohnheitsverbrecher`, but I think it's also fine as is. 
```suggestion
In seltenen Umständen können Support-Tickets von Gewohnheitsverbrechern absichtlich ignoriert werden, jedoch wird sich das [Support-Team](/wiki/People/The_Team/Account_support_team) vorher **immer** bei dir melden, wenn sie nicht mehr länger antworten werden. Falls du noch keine Benachrichtigung wie diese erhalten hast, brauchst du dir keine Sorgen machen!
```

Comment (FamousPig) [reply wiki/Help_centre/Website/de.md:34]: Easiest translation would be `wiederholt straffällig` I think. Or maybe offending is meant to mean `beleidigend`/insulting in this context? As in you don't get ignored for breaking the rules of the game, you get ignored for being rude to the support team...

Comment (The-Last-Cookie) [reply wiki/Help_centre/Website/de.md:34]: I think so too. The phrase refers to people that are rude and insulting to the support team several times. I will leave the translation as-is 👍

**Review by respektive — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**